open_instruct/grpo_fast.py

```diff
@@ -610,7 +614,10 @@ def from_pretrained(
         self.policy.gradient_checkpointing_enable()
         # AdamOptimizer = DeepSpeedCPUAdam if self.adam_offload else FusedAdam
         # AdamOptimizer = FusedAdam
-        optim_params = get_optimizer_grouped_parameters(self.policy, args.weight_decay)
+        if args.set_weight_decay_on_bias_and_norm:
```

Comment:
yay!

Reply:
yeah let's have this option. was debugging something here and there.